src/test/java/com/google/sps/ChatServletTest.java

```diff
@@ -34,12 +34,18 @@
 
 @RunWith(JUnit4.class)
 public final class ChatServletTest extends Mockito {
-    private static final String GROUP_ID = "123";
+    private static final String BLACK_LIVES_MATTER = "Black Lives Matter";
+    private static final String BLM_GROUP_ID = "123";
     private static final String MESSAGE_TEXT_NON_TOXIC = "hello";
     private static final String MESSAGE_TEXT_TOXIC = "what kind of idiot name is foo?";
+    private static final String SIERRA_CLUB = "Sierra Club";
+    private static final String SIERRA_GROUP_ID = "456";
     
-    private static final ImmutableList<String> SAMPLE_GROUP_LIST = 
-        ImmutableList.of("Black Lives Matter");
+    private static final ImmutableList<ImmutableMap<String, String>> SAMPLE_GROUP_LIST = 
+        ImmutableList.of(ImmutableMap.of(GROUP_NAME_PROPERTY, "Black Lives Matter",
```

Comment:
For readability, separate these out into GROUPs
so
GROUP_1 = immutableMap.of(....);
GROUP_2 = immutableMap.of(...);

SAMPLE_GROUP_LIST = ImmutableList.of(GROUP_1, GROUP_2);